doc/design/eventdriven-callbacks.md

```diff
@@ -0,0 +1,171 @@
+## Name proposals
+
+ - ReactAl = React And Listen
```

Comment:
With `React And Listen` the causality is violated and our universe might implode. This is more a `Listen and React` -> `LiAR`

Reply:
Reactor makes real sense since we use the Reactor Pattern behind the scenes.

Reply:
What about uncreative name CallSet to be somehow consistent to WaitSet?!

Reply:
Yeah, why is it capitalized like this (with the small L) ? It looks a bit weird to me.
Also, I keep reading it as "Rectal"...

Reply:
@elfenpiff I have another one your definitely like `event listening for execution` -> `ELFE`